### PR TITLE
Inclusion of ES6

### PR DESCRIPTION
**Pull request by Kirabo19**
Re-create  Awesome books app as detailed below :

- Use ES6 syntax.
- Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file.

### CODE REVIEW

File: modules/ShowContact.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export default function showContact() {
+  const openContact = document.getElementById('open-contact');
+  const addNew = document.getElementById('add_books');
+  const bookList = document.getElementById('books-list');
```
Comment (thecodechaser) [inline modules/ShowContact.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: modules/addBook.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import Book from './app.js';
+
+// const book = new Book(title, author);
+
+export default function addBooks() {
+  const form = document.querySelector('#books-form');
+
+  form.addEventListener('submit', () => {
```
Comment (thecodechaser) [inline modules/addBook.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: modules/bookList.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export default function showBook() {
+  const addBookToList = document.getElementById('bk-List');
+  const addNew = document.getElementById('add_books');
+  const bookList = document.getElementById('books-list');
```
Comment (thecodechaser) [inline modules/bookList.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: modules/showForm.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export default function showForm() {
+  const openForm = document.getElementById('add-form');
+  const addNew = document.getElementById('add_books');
+  const bookList = document.getElementById('books-list');
+  const contact = document.getElementById('contact');
```
Comment (thecodechaser) [inline modules/showForm.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: modules/app.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+// function showContact() {
+//   contact.classList.remove('dontShow');
+//   bookList.classList.add('dontShow');
+//   addNew.classList.add('dontShow');
+// }
+
+// openForm.addEventListener('click', showForm);
+
+// addBookToList.addEventListener('click', showBook);
```
Comment (thecodechaser) [inline modules/app.js:110]: - [ ] Please remove commented code from all of your files, This will make sure that your code looks clean and professional.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Kirabo19👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🍾🎉**
Hello @Kirabo19 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**